types/mithril/index.d.ts

```diff
@@ -3,7 +3,8 @@
 // Definitions by: Mike Linkovich <https://github.com/spacejack>
 //                 Claudia Meadows <https://github.com/dead-claudia>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-// TypeScript Version: 3.2
+// TypeScript Version: 5.0
+// Minimum TypeScript Version: 5.3
```

Comment:
This is where the error's coming from, BTW.

Reply:
Fixed! Didn't know that counts too!